policyengine_us/variables/gov/states/ma/doer/liheap/ma_liheap_utility_category.py

```diff
@@ -0,0 +1,52 @@
+from policyengine_us.model_api import *
+
+
+class MassachusettsLIHEAPUtilityCategory(Enum):
+    UTILITY_AND_HEAT_IN_RENT = "Utility and Heat in Rent"
+    DELIVERABLE_FUEL = "Deliverable Fuel"
```

Comment:
Can we add a default None value here - for filers who don't meet either criteria